src/lib/operators/aggregate_sort.cpp

```diff
@@ -582,9 +684,15 @@ std::shared_ptr<const Table> AggregateSort::_on_execute() {
     aggregate_index++;
   }
 
-  // Append output to result table
   result_table->append_chunk(_output_segments);
 
+  const auto chunk = result_table->last_chunk();
+  auto sorted_by = sorted_table->get_chunk(ChunkID{0})->sorted_by();
+  if (sorted_by) {
+    chunk->finalize();
+    chunk->set_sorted_by(*sorted_by);
+  }
+
```

Comment:
This looks weird to me.
* It is not guaranteed that the `sorted_by` column of the input is part of the output or is at a different position?
* Why doesn't this check if we actually grouped by that column?
* What happens if we have input value clustering?

Reply:
As the sort operator requires "consecutiveness" and not sortedness, I would remove setting the flag. The only situation where we can set it without checking various condidtions is when the whole input needed to be sorted. As this is the fall back situation where the AggregateSort performs worst, I don't see a value to setting the sorted information in this case.
We could set the value clustered flag, but we generate a single chunk so I don't see any advantages there either.

What's your opinion on that @j-tr , @Boffmann , @MasterCarl ?

Reply:
I would change it to:

```
  result_table->append_chunk(_output_segments);

  // We do not set sort information on the output table as we can only guarantee it in certain situations (e.g., when
  // the whole input table needed to be sorted). As this aggregate case is the fall back solution and should be rather
  // be calculated with the hash aggregate, we neglect the sort information for now and keep it simple.
  return result_table;
```